.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Prepare a new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      strategy:
+        description: Valid semver number <x.x.x> or strategy <patch, minor, major>
+        default: "patch"
+        required: false
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
```

Comment:
You may want to add an `if` statement on `github.actor` : If I remember correctly, `workflow_dispatch` can be triggered by all collaborators, not just maintainers.

Reply:
Done
Only @rande @legullec @fthouraud and me could dispatch this workflow. (sorry Olivier I do not have your github user name :) )